inst/extdata/energy/A44.flsp_bm2_state_res.csv

```diff
@@ -0,0 +1,56 @@
+# File: A44.flsp_bm2_state_res.csv
+# Title: Residential floorspace by state
+# Units: Billion m2
+# Source: Written by L111_Res_flsp.R; RGCAM output
```

Comment:
This is certainly confusing. I second Marshall's request to add appropriate comments.